Symfony/CS/Fixer/Contrib/CombineConsecutiveUnsetsFixer.php

```diff
@@ -83,8 +83,8 @@ public function getDescription()
      */
     public function getPriority()
     {
-        // should ran before SpacesAfterSemicolonFixer, WhitespacyLinesFixer, TrailingSpacesFixer and ExtraEmptyLinesFixer.
-        return 25;
+        // should ran before SpacesAfterSemicolonFixer, WhitespacyLinesFixer, TrailingSpacesFixer and ExtraEmptyLinesFixer and after NoEmptyStatementFixer.
+        return 24;
```

Comment:
Interesting idea, I put it on my backlog of things to give a go ;)

For now I'm going to have a look at the priority numbers and put some more space between those. The values themselves are pretty arbitrary, however having more space between those will make adding other fixers more easy (until we ran out of space with those in ;)), this also helps people fitting in custom fixers more easy.

Beside the priority numbers I would like to do a bit more testing. Like to hear back from @gharlan as he spotted an issue with the fixer before :)